Add divsen@ profile (#3917)
opensearch-project/project-website: _community_members/divsen.md

---
short_name: 'divsen'
name: 'Divyam Sengar'
photo: '/assets/media/community/members/divyamsengar.png'

> job_title_and_company: 'Software Development Engineer Intern at AWS - OpenSearch'

Reviewer: ```suggestion
job_title_and_company: 'Software Development Engineer Intern at AWS -- OpenSearch'
```

primary_title: 'Divyam Sengar'
title: 'OpenSearch Community Member: Divyam Sengar'
breadcrumbs:
  icon: community
  items:
    - title: Community
      url: /community/index.html
    - title: Members
      url: /community/members/index.html
    - title: "Divyam Sengar&apos;s Profile"
      url: '/community/members/divyam-sengar.html'
github: "DivyamSengar"
linkedin: "divyam-sengar"
permalink: '/community/members/divyam-sengar.html'
personas:
  - conference_speaker
  - author
---

> **Divyam Sengar** is a software development engineer intern at AWS working on the OpenSearch Project. He works on Automation, AI Infrastructure, & OSCAR for OpenSearch, leading Agentic AI into simplifying tasks for OpenSearch.

Reviewer: ```suggestion
**Divyam Sengar** is a software development engineer intern at AWS working on the OpenSearch Project. He works on automation, AI infrastructure, and OSCAR for OpenSearch.
```
